source/tutorial/swiftui.txt

```diff
@@ -21,12 +21,20 @@ the creation of a to-do list application. This application enables users to:
 
 - Register their email as a new user account.
 - Sign in to their account with their email and password (and sign out later).
-- View, create, modify, and delete tasks.
+- View, create, modify, and delete their own tasks.
+- View all tasks, even where the user is not the owner.
 
-This tutorial adds on to the Template App. You will add a new ``priority`` field 
+The template app also provides a toggle that simulates the device being in 
+"Offline Mode." This toggle lets you quickly test Device Sync functionality 
+on a simulator, simulating the user having no internet connection. However,
+you would likely remove this toggle in a production application.
+
+This tutorial builds on the Template App. You will add a new ``priority`` field 
 to the existing ``Item`` model and update the 
 :ref:`Flexible Sync subscription <queryable-fields>` to only show items within 
-a range of priorities. 
+a range of priorities. This is an arbitrary example to illustrate how you might 
+adapt the template app for your own needs. You would not realistically make 
+this specific change given the current structure of the template app. 
```

Comment:
like i agree w you here, but i think we could *soften* the language to smthn like:

```suggestion
a range of priorities. This example illustrates how you might 
adapt the template app for your own needs. You would not necessarily make 
this change given the current structure of the template app. 
```